src/lib/helpers/styles.js

```diff
@@ -0,0 +1,75 @@
+export const styles = {
```

Comment:
This can be renamed to jsonEditorDefaultStyles, see comment below

Reply:
Resolved by renaming to jsonEditorDefaultStyles